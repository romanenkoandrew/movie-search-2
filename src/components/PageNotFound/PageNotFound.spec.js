import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './PageNotFound';

describe('PageNotFound component', () => {
  it('should render PageNotFound component', () => {
    const component = shallow(<PageNotFound />);
    expect(component).toMatchSnapshot();
  });
});
