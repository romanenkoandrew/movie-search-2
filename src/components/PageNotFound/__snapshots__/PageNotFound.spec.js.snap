// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound component should render PageNotFound component 1`] = `
<Result
  extra={
    <ForwardRef(InternalButton)
      block={false}
      ghost={false}
      htmlType="button"
      loading={false}
      type="primary"
    >
      <ForwardRef
        to="/"
      >
        Back Home
      </ForwardRef>
    </ForwardRef(InternalButton)>
  }
  status="404"
  subTitle="Sorry, the page you visited does not exist."
  title="404"
/>
`;
